python_files/vscode_pytest/run_pytest_script.py

```diff
@@ -17,10 +16,13 @@
 script_dir = pathlib.Path(__file__).parent.parent
 sys.path.append(os.fspath(script_dir))
 sys.path.append(os.fspath(script_dir / "lib" / "python"))
-from testing_tools import (  # noqa: E402
-    process_json_util,
-    socket_manager,
-)
+
+
+def run_without_test_ids(args):
+    print("Running pytest with no test ids as args. Args being used: ", args)
+    arg_array = ["-p", "vscode_pytest", *args]
+    pytest.main(arg_array)
```

Comment:
```suggestion
def run_pytest(args):
    arg_array = ["-p", "vscode_pytest", *args]
    pytest.main(arg_array)
```
I think all the cases below should just call this. This way we don't duplicate passing `-p vscode_pytest`. Also there is really no difference is how `pytest.main` is invoked with or without ids.

Reply:
wdym about the pytest.main comment? Like that the two invocations are the same but the test_ids when included do still impact?